kumascript/src/info.js

```diff
@@ -229,8 +229,8 @@ const info = {
           });
           if (!summary) {
             // To avoid <p> tags that are inside things like
-            // `<div class="blockIndicator>`, just remove those divs first.
-            $("div.blockIndicator, div.note").remove();
+            // `<div class="notecard>`, just remove those divs first.
+            $("div.notecard, div.note").remove();
```

Comment:
I think we still need to support the old stuff (i.e., `div.blockIndicator`) as well, since there seems to be lots of those sprinkled among the documents apart from being generated by macros. cc'ing @peterbe for his thoughts here as well.
```suggestion
            $("div.notecard, div.note, div.blockIndicator").remove();
```

Reply:
Once we launch Yari and the `mdn/content` repository becomes source of truth for the content, we can clean-up/remove these deprecated CSS classes like `blockIndicator`.